src/js/ProductList.mjs

```diff
@@ -23,12 +23,16 @@ export default class ProductList {
   async init() {
     // Get the data from the data source
     const list = await this.dataSource.getData();
-    
+
     // Get unique products based on Id
-    const uniqueProducts = this.getUniqueProducts(list);
+    // const uniqueProducts = this.getUniqueProducts(list);
```

Comment:
I'm not sure why this unique function is needed... there aren't any duplicates products. There are very similar products, but not duplicates. You'll be moving away from this JSON file with the next activity and using a real server to get products.